docs/pilot.md

```diff
@@ -58,38 +57,38 @@ The container image can be used directly by Singularity (no prior download requi
    ```
  * It is possible that you see some scary looking warnings, but those can be ignored for now.
 
-   To verify that things are working, check the contents of the `/cvmfs/pilot.eessi-hpc.org/2020.09` directory:
+   To verify that things are working, check the contents of the `/cvmfs/pilot.eessi-hpc.org/2020.10` directory:
    ```shell
-   Singularity> ls /cvmfs/pilot.eessi-hpc.org/2020.09
+   Singularity> ls /cvmfs/pilot.eessi-hpc.org/2020.10
    compat  init  software
    ```
 ### Setting up the EESSI environment
 
 Once you have the EESSI pilot repository mounted, you can set up the environment by sourcing the provided init script:
 
 ```shell
-source /cvmfs/pilot.eessi-hpc.org/2020.09/init/bash
+source /cvmfs/pilot.eessi-hpc.org/2020.10/init/bash
 ```
 
 If all goes well, you should see output like this:
 
 ```shell
-Singularity> source /cvmfs/pilot.eessi-hpc.org/2020.09/init/bash
-Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/2020.09!
-Derived subdirectory for software layer: x86_64/intel/cascadelake
+Singularity> source /cvmfs/pilot.eessi-hpc.org/2020.10/init/bash
+Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/2020.10!
+Derived subdirectory for software layer: x86_64/amd/zen2
 Initializing Lmod...
-Prepending /cvmfs/pilot.eessi-hpc.org/2020.09/software/x86_64/intel/cascadelake/modules/all to $MODULEPATH...
+Prepending /cvmfs/pilot.eessi-hpc.org/2020.10/software/x86_64/amd/zen2/modules/all to $MODULEPATH...
 
-*** Known problems in the 2020.09 pilot software stack ***
+*** Known problems in the 2020.10 pilot software stack ***
 
 1) No Lmod cache has been generated yet for the module files included
-   in the 2020.09 pilot software stack.
+   in the 2020.10 pilot software stack.
    As a result, 'module' commands may be somewhat slow.
    (see also https://github.com/EESSI/software-layer/issues/10)
 
```

Comment:
Should we mention the GCC/ld issue here too?
https://github.com/EESSI/software-layer/issues/26